chia/full_node/mempool.py

```diff
@@ -55,6 +62,14 @@ def remove_from_pool(self, item: MempoolItem):
         self.total_mempool_cost -= item.cost
         assert self.total_mempool_cost >= 0
 
+        mempool_info = FeeMempoolInfo(
+            uint64(self.max_size_in_cost),
+            uint64(self.minimum_fee_per_cost_to_replace),
+            uint64(self.total_mempool_cost),
+            datetime.now(),
+        )
```

Comment:
Minor suggestion -- add a `feeSample()` method to create the `FeeMempoolInfo` instance, since this logic duplicated in `add_to_pool()`.

Reply:
Also totally minor, but I was wondering if there's utility in having the caller provide the timestamp, or whether it would be better to create that in the FeeMempoolInfo constructor.